packages/zoe/src/contracts/simpleExchange.js

```diff
@@ -95,29 +95,46 @@ const start = zcf => {
     return counterOffers;
   }
 
+  const sellAssetForPrice = harden({
+    give: { Asset: null },
+    want: { Price: null },
+  });
+
+  const sell = seat => {
+    buySeats = swapIfCanTradeAndUpdateBook(buySeats, sellSeats, seat);
+    bookOrdersChanged();
```

Comment:
You should move the call `bookOrdersChanged();` into `swapIfCanTradeAndUpdateBook`. That way there's no hazard of someone calling `swapIfCanTradeAndUpdateBook` and forgetting to notify.

Reply:
Good idea